src/main/java/io/supertokens/webserver/api/useridmapping/UserIdMappingAPI.java

```diff
@@ -149,7 +149,7 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws IO
 
         try {
             io.supertokens.pluginInterface.useridmapping.UserIdMapping userIdMapping = UserIdMapping
-                    .getUserIdMapping(this.getTenantIdentifier(req), main, userId,
+                    .getUserIdMapping(this.getTenantIdentifier(req).toAppIdentifier(), main, userId,
```

Comment:
so here we call the StorageLayer.getUserForStorageAndResolveUserIdMapping

Reply:
Done